internal/services/helpers.go

```diff
@@ -134,3 +136,25 @@ func ReturnErr(w http.ResponseWriter, err string, code int) {
 	w.WriteHeader(code)
 	json.NewEncoder(w).Encode(errj)
 }
+func CheckEmail(w http.ResponseWriter, email string) {
+	emailRegex := regexp.MustCompile(`^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}$`)
+	if !emailRegex.MatchString(email) {
+		ReturnErr(w, "Invalid email format", http.StatusBadRequest)
+		return
+	}
+	if len(email) > 50 {
```

Comment:
I'm pretty sure it can be more than 50 
![image](https://github.com/mo-pm888/twitter/assets/1732280/ff727804-4033-48cd-83b9-f2778e80ef59)